src/pages/Designer/Designer.js

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import {
+  Grommet,
+  Box,
+  Heading,
+  Text,
+  Stack,
+  Paragraph,
+  Image,
+  Main,
+} from 'grommet';
+import {
+  ClearOption,
+  Configure,
+  Camera,
+  CloudUpload,
+  Selection,
+  BarChart,
+} from 'grommet-icons';
+import { grommetToolsTheme } from '../../theme.js';
+import AppHeader from '../../components/AppHeader.js';
+import IconCard from '../../components/IconCard.js';
+
+const Designer = () => (
```

Comment:
This component has so many `margin={{ left: 'xlarge' }}`it makes me thing the margin should be placed on a more generic parent layout.

Reply:
was this addressed in any way?

Reply:
Yes I was able to use the margin on a parent component in some spots but not in all cases. Notably on the icons below the large designer icon I wasn't able to get the spacing right with margin on a parent box